cli/tox.ini

```diff
@@ -23,11 +23,11 @@ commands =
 
 [testenv:py27-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
 
 [testenv:py34-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
```

Comment:
Okay, I'll change it to `-x` since we should be running tests locally before pushing PRs.